CHANGES

```diff
@@ -4,7 +4,44 @@ Release 2.0.0 (in development)
 Dependencies
 ------------
 
-* LaTeX builder now depends on TeX Live 2015 or above
+* LaTeX builder now depends on TeX Live 2015 or above.
+* LaTeX builder may need these additional LaTeX packages for PDF builds
+  (with ``'pdflatex'`` :confval:`latex_engine`):
+
+  .. list-table:: Requirements
+     :header-rows: 1
+
+     * - LaTeX package (CTAN)
+       - Ubuntu xenial
+       - needed for
+     * - substitutefont
+       - texlive-latex-extra
+       - Greek or Cyrillic letters (in non-Cyrillic documents)
+     * - textalpha
+       - texlive-lang-greek
+       - Greek letters (in text, not math)
+     * - gfsartemisia
+       - fonts-gfs-artemisia (texlive-fonts-extra)
+       - Greek letters (in text, not math)
+     * - gfsneohellenic
+       - fonts-gfs-neohellenic (texlive-fonts-extra)
+       - Greek letters (in text, not math)
+     * - cbfonts
+       - texlive-lang-greek
+       - Greek letters (in text, not math)
+     * - cm-lgc
+       - texlive-fonts-extra
+       - Cyrillic letters (in non-Cyrillic documents)
```

Comment:
The X2 encoded Computer Modern seem to require not only `texlive-lang-cyrillic` but also [`cm-super-minimal`](https://packages.ubuntu.com/xenial/cm-super-minimal) (because PDF needs subsetting in particular font file `sfrm1000.pfb`, if I look at log of `pdflatex` at my locale). Investigating if `cm-super-minimal` is enough or `cm-super` is needed.

Reply:
>  Investigating if cm-super-minimal is enough or cm-super is needed.

Yes, cm-super-minimal is enough, but `\usepackage[10pt]{type1ec}` must be used before `\usepackage[...]{fontenc}`. From the README.debian of the `cm-super-minimal` package, 

>  If you DON'T have cm-super (the full package) installed you have to use
>       \usepackage[10pt]{type1ec}
>  which uses only those cm-super fonts available in cm-super-minimal and
>  scale those fonts for other design sizes.

and I confirmed by looing at contents of `type1ec.sty` file to see what it does.